src/Command/Processor/ProcessorChain.php

```diff
@@ -93,7 +93,7 @@ public function count()
     }
 
     /**
-     * {@inheritdoc}
+     * @return bool
```

Comment:
These new blocks are missing `@param`.

Reply:
well, ArrayAccess only allows mixed for them, how do you want to document this? `@param mixed`?

Reply:
Yes, we can use mixed for everything for now.

Reply:
Actually I went with `@param int` as done in Cluster/SlotMap